.ubidoc.yml

```diff
@@ -0,0 +1,27 @@
+tables:
+  - name: "milk-planning-ul"
+    termName: "Term"
+    definitionName: "Definition"
+    rows:
+      - class: "ProcessedMilk"
+      - class: "QuintalsOfMilk"
+      - class: "Yield"
+      - type: "RecipeBook"
+      - type: "Stock"
+      - class: "StockedQuantity"
+      - class: "Quantity"
+      - class: "RequestedProduct"
+
+  - name: "milk-planning-incoming"
+    termName: "Event"
+    definitionName: "Description"
+    rows:
+      - case: "ReceivedOrder"
```

Comment:
Remember that, in the new plugin version (once we merge my open PR) you will have to specify the full case name

Reply:
I don't quite understand what you mean. In the new versione I have to specify something like this:
```
rows:
  - case: "IncomingEvents.ReceivedOrder`
```

Reply:
Yes, exactly; also if you don't like the full name you can specify a custom name:
```yaml 
rows: 
  - case: "IncomingEvent.ReceivedOrder`
    name: "Received Order"
```